core/src/main/java/net/sourceforge/jnlp/util/logging/FileLog.java

```diff
@@ -50,26 +51,48 @@
 
     private static final Logger LOG = LoggerFactory.getLogger(FileLog.class);
 
-    private static final String TIME_SEPARATOR = OsUtil.isWindows() ? "_" : ":";
+    public static String LOG_PREFIX_ENV = "itwLogFilePrefix";
+    public static String LOG_POSTFIX_ENV = "itwLogFilePostfix";
+
     private static final String DEFAULT_LOGGER_NAME = TextsProvider.ITW + " file-logger";
+    private static final String TIME_SEPARATOR = OsUtil.isWindows() ? "_" : ":";
     private static final SimpleDateFormat FILE_LOG_NAME_FORMATTER = new SimpleDateFormat("yyyy-MM-dd_HH" + TIME_SEPARATOR + "mm" + TIME_SEPARATOR + "ss.S");
 
-    public static Header getHeadlineHeader() {
-        return new Header(OutputControllerLevel.WARN, false);
+    private static final String logFileNamePrefix;
+    private static String logFileNamePostfix;
+
+    static {
+        final String envPrefix = System.getenv(LOG_PREFIX_ENV);
+        if (!StringUtils.isBlank(envPrefix)) {
+            logFileNamePrefix = envPrefix;
+        } else {
+            logFileNamePrefix = FILE_LOG_NAME_FORMATTER.format(new Date());
+        }
+
+        final String envPostfix = System.getenv(LOG_POSTFIX_ENV);
+        if (!StringUtils.isBlank(envPostfix)) {
+            logFileNamePostfix = envPostfix;
+        } else {
+            logFileNamePostfix = "itw";
```

Comment:
we already log in an itw folder. Do we really need as a prefix, too?

Reply:
what else would you do as a postfix???